tools/pctools/venv_inst.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
Perhaps it's a good idea to fail the bash script in case of errors in the commands?

```suggestion
#!/bin/bash -e
```